vizier/datastore/mimir/dataset.py

```diff
@@ -152,7 +151,7 @@ def to_sql_value(self, value):
         # convert the given argument to check whether it actually is a numeric
         # value. Note that we always return a string beacuse the result is
         # intended to be concatenated as part of a SQL query string.
-        if self.is_numeric():
+        if self.data_type.lower() in ['int', 'real']:
```

Comment:
Especially since this is a hardcoded list, it would be useful to wrap it in a utiltiy function (like is_numeric()).

Reply:
vizier/engine/packages/base.py contains a list of Mimir types, why not use those here?